File: Task-03/C/Subtask-3.c

#include <stdio.h>
#include <math.h>
void main() {
int row;
printf("enter no. of rows:");
scanf("%d",&row);
int x = floor(row/2);
for(int i =1;i<=x+1;i++){
    for(int j =1;j<=(row-i)-2;j++){
        printf(" ");
            }
    for(int j =1;j<=(2*i)-1;j++){
        printf("*");
            }
    printf("\n");
        }
for(int i= x;i>=0;i--){
    for(int j =1;j<=(row-i)-2;j++){
        printf(" ");
            }
    for(int j =1;j<=(2*i)-1;j++){
        printf("*");
            }
    printf("\n");
        }


}

File: Task-03/Pyhton/Subtask-3.py
n=int(input("enter number of rows"))
x=int(n/2+0.5) 
for i in range(1,x+1): # first half 
    for j in range((n-i)-2):   # spaces
        print(" ",end="")
    for j in range(1,2*i):   #stars   
        print("*",end="")
    print() #next line after each row
for i in range(x-1,0,-1): #2nd half
    
    for j in range((n-i)-2):   
        print(" ",end="")
    for j in range(1,2*i):      
        print("*",end="")
    print()